# Odometry data
nav_msgs/Odometry odometry

# Occupancy grid map
nav_msgs/OccupancyGrid occupancy_grid

# User position
geometry_msgs/PoseStamped user_position

# Centroid point cloud
sensor_msgs/PointCloud2 centroid_point